Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
>  it needed to be in init because that guarantees toolbarItems isn't empty, which allows the toolbar to be shown.

As far as I know, toolbar visibility isn't determined by the toolbar items, it is determined by `setToolBarHidden`. Do you have documentation saying that the toolbar will ignore `toolBarHidden=YES` if `toolbarItems` are nil?